src/Chapter2.hs

```diff
@@ -832,7 +842,8 @@ list.
 
 🕯 HINT: Use the 'cycle' function
 -}
-rotate = error "rotate: Not implemented!"
+rotate :: Int -> [a] -> [a]
+rotate n xs = drop n $ take (length xs + n) (cycle xs)
```

Comment:
That's a neat implementation :ok_hand: 
Though, it can be improved for large `n`s. Note, that if you're a given a list of size 5, and you need to rotate it one million times, your implementation will be a bit slow. But nevertheless, kudos for writing clean code!

Also, I noticed that it doesn't work properly on negative numbers, and doesn't return empty list on negative `n`. Corner cases can be sometimes tricky to see and handle properly...